### PR TITLE
docs: update CHANGELOG for v2.2

### PR DESCRIPTION
**Pull request by membphis**
Please answer these questions before submitting a pull request

- Why submit this pull request?
- [ ] Bugfix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bugfix
- Description

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,18 +19,46 @@
...
 
+- [2.2.0](#220)
 - [2.1.1](#211)
 - [2.1.0](#210)
 - [2.0.0](#200)
 - [1.5.0](#150)
 - [1.0.0](#100)
 
 
+# 2.2.0
```
Comment (membphis) [inline CHANGELOG.md:30]: Should we use `2.2` here?

Comment (juzhiyuan) [reply CHANGELOG.md:30]: ok for me, cc @nic-chen

Comment (nic-chen) [reply CHANGELOG.md:30]: I think this is OK now, we have been using it like this.
https://raw.githubusercontent.com/apache/apisix/master/CHANGELOG.md

**Review by LiteSun — APPROVED** [review]
It ok for me, except Check markdown links CI failed

**Review by nic-chen — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Pull request merged**